packages/ui-react/src/components/LanguageMenu/LanguageMenu.test.tsx

```diff
@@ -24,21 +24,29 @@ describe('<LanguageMenu>', () => {
     expect(container).toMatchSnapshot();
   });
 
-  test('shows all languages in the menu', () => {
+  test('shows all languages in the menu when button is clicked and calls the onClick callback with the correct language code', () => {
```

Comment:
I like your addition, but a non-technical description is usually preferably (like from a users perspective), such as:
```suggestion
  test('shows all languages in the menu when button is clicked and handles clicks using the correct language code', () => {
```

Reply:
- [x] updated test description to be more user friendly.